pytest_tests/testsuites/failovers/test_failover_service_kill.py

```diff
@@ -0,0 +1,334 @@
+from python_keywords.node_management import check_node_in_map, storage_node_healthcheck
+from tenacity import retry, stop_after_attempt, wait_fixed
+from test_control import wait_for_success
+from wallet import WalletFactory, WalletFile
+from wellknown_acl import PUBLIC_ACL
+
+from steps.cluster_test_base import ClusterTestBase
+
+logger = logging.getLogger("NeoLogger")
+
+NEOFS_IR = "neofs-ir"
+NEOFS_HTTP = "neofs-http"
+NEOFS_NODE = "neofs-storage"
+NEO_GO = "neo-go"
+
+
+@pytest.mark.failover
+@pytest.mark.failover_network
+class TestFailoverServiceKill(ClusterTestBase):
+    PLACEMENT_RULE = "REP 1 IN X CBF 1 SELECT 1 FROM * AS X"
+
+    @pytest.fixture(scope="class")
+    def user_wallet(self, wallet_factory: WalletFactory) -> WalletFile:
+        return wallet_factory.create_wallet()
+
+    @pytest.fixture(scope="class")
+    @allure.title("Create container")
+    def user_container(self, user_wallet: WalletFile):
+        return create_container(
+            wallet=user_wallet.path,
+            shell=self.shell,
+            endpoint=self.cluster.default_rpc_endpoint,
+            rule=self.PLACEMENT_RULE,
+            basic_acl=PUBLIC_ACL,
+        )
+
+    @allure.title("Check and return  status of given service")
+    def service_status(self, service: str, shell: Shell) -> str:
+        return shell.exec(f"sudo systemctl is-active {service}").stdout.rstrip()
+
+    @allure.step("Run health check to node")
+    def health_check(self, node: StorageNode):
+        health_check = storage_node_healthcheck(node)
+        assert health_check.health_status == "READY" and health_check.network_status == "ONLINE"
+
+    @allure.step("Wait for expected status of passed service")
+    @wait_for_success(60, 5)
+    def wait_for_expected_status(self, service: str, status: str, shell: Shell):
+        real_status = self.service_status(service=service, shell=shell)
+        assert (
+            status == real_status
+        ), f"Service {service}: expected status= {status}, real status {real_status}"
+
+    @allure.step("Run neo-go dump-keys")
+    def neo_go_dump_keys(self, shell: Shell, node: StorageNode) -> dict:
+        host = node.host
+        service_config = host.get_service_config(node.name)
+        wallet_path = service_config.attributes["wallet_path"]
+        output = shell.exec(f"neo-go wallet dump-keys -w {wallet_path}").stdout
+        try:
+            # taking first line from command's output contain wallet address
+            first_line = output.split("\n")[0]
+        except Exception:
+            logger.error(f"Got empty output (neo-go dump keys): {output}")
+        address_id = first_line.split()[0]
+        # taking second line from command's output contain wallet key
+        wallet_key = output.split("\n")[1]
+        return {address_id: wallet_key}
+
+    @allure.step("Run neo-go query height")
+    def neo_go_query_height(self, shell: Shell, node: StorageNode) -> dict:
```

Comment:
In a testlib
https://github.com/TrueCloudLab/frostfs-testlib/blob/980158ad72180f93d6e7acfc05946faa9feba21c/src/neofs_testlib/cli/neogo/query.py#L44